bus-mapping/src/evm/opcodes.rs

```diff
@@ -283,7 +283,7 @@ pub fn gen_associated_ops(
         );
 
         exec_step.error = Some(exec_error);
-        if geth_step.op.is_call_or_create() {
+        if geth_step.op.is_call_or_create() && !exec_step.oog_or_stack_error() {
```

Comment:
it would be better to add several lines of comments here to explain why oog and stack don't need the push_call

Reply:
added in [aa107c0](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/674/commits/aa107c0271feb0aa70118b0d596dcb34304d9afd) thanks!!